packages/block-library/src/navigation/edit/index.js

```diff
@@ -655,6 +655,19 @@ function Navigation( {
 		</InspectorControls>
 	);
 
+	const ManageMenusButton = ( { buttonClassName = '' } ) => (
+		<Button
+			variant="link"
+			disabled={ ! hasManagePermissions || ! hasResolvedNavigationMenus }
```

Comment:
These could be provided as props. This would make the component more isolated from it's environment and make it easier to extract to a separate file in future.

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/45782